packages/connect-webextension/src/proxy/index.ts

```diff
@@ -0,0 +1,134 @@
+import EventEmitter from 'events';
+
+// NOTE: @trezor/connect part is intentionally not imported from the index due to NormalReplacementPlugin
+// in packages/suite-build/configs/web.webpack.config.ts
```

Comment:
you sure about `packages/suite-build/configs/web.webpack.config.ts`?

Reply:
https://github.com/trezor/trezor-suite/pull/10751/commits/c294594cbcbc44bc391611dccb010090434d9de3